substrate/frame/revive/src/precompiles.rs

```diff
@@ -25,7 +25,7 @@
 //!
 //! Use `alloy` through our re-export in this module to implement Eth ABI.
 
-mod builtin;
+pub mod builtin;
```

Comment:
No don't make this whole thing public. Just re-export the interface if you need it.